decko/lib/decko/config/environments/development.rb

```diff
@@ -80,7 +80,7 @@
 
   config.action_mailer.perform_deliveries = false
 
-
+  config.active_job.queue_adapter = :inline
```

Comment:
default was `:async`.  that's not a good mix with `allow_concurrency = false`, especially in rake tasks.

Reply:
This kills our migrations because after a first card change in a migration the whole card namespace is gone including `Card::Auth.current_id`. It seems to work with `config.active_job.queue_adapter = :delayed_job`.

Reply:
wow.  any idea why :inline would make us lose the card namespace??